File: c/03.loops/Armstrong.c
#include<stdio.h>
int main()
{
	int num,num1,rev,ans;

	printf("Enter the number : ");
	scanf("%d",&num);

	num1=num;
	ans=0;

	while(num>0)
	{
		rev=num%10;
		ans=ans+(rev*rev*rev);
		num=num/10;
	}
	if(num1==ans)
	{
	printf("Given number %d is Armstrong !!",ans);
	}
	else
	{
		printf("Given number %d is not Armstrong !!",ans);
	}
	return 0;
}

File: python/1. Fundamentals/armstrong.py
num=int(input("Enter the number : "))

tnum=num
sum=0
while num>0:
	rem=num%10
	sum=sum+(rem**3)
	num=num//10

if sum==tnum:
	print(f'{tnum} is the armstrong number')
else:
	print(f'{tnum} is not a armstrong number')

